File: VB/Q147080/Tab/Styles/MyWindowsXPViewInfoRegistrator.vb
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.XtraTab.ViewInfo
Imports DxSample.Tab.ViewInfo

Namespace DxSample.Tab.Styles
    Public Class MyWindowsXPViewInfoRegistrator
        Inherits WindowsXPViewInfoRegistrator

        Public Sub New()
            MyBase.New()
        End Sub
        Public Overrides Function CreateHeaderViewInfo(ByVal viewInfo As BaseTabControlViewInfo) As BaseTabHeaderViewInfo
            Return New MyWindowsXPTabHeaderViewInfo(viewInfo)
        End Function
    End Class
End Namespace

File: CS/Q147080/Tab/Styles/MyWindowsXPViewInfoRegistrator.cs
using DevExpress.XtraTab.Registrator;
using DevExpress.XtraTab.ViewInfo;
using DxSample.Tab.ViewInfo;

namespace DxSample.Tab.Styles {
    public class MyWindowsXPViewInfoRegistrator : WindowsXPViewInfoRegistrator {
        public MyWindowsXPViewInfoRegistrator() : base() { }
        public override BaseTabHeaderViewInfo CreateHeaderViewInfo(BaseTabControlViewInfo viewInfo) {
            return new MyWindowsXPTabHeaderViewInfo(viewInfo);
        }
    }
}
